docs/digger/installation/macos-variables.asciidoc

```diff
@@ -30,17 +26,14 @@ This installation step will also add any taps and packages specified in the opti
 |A list of taps to add.
 |===
 
-== Install Ruby
+=== Install Ruby
 
-To run this step individually use `osx_install_ruby` tag (see <<Running Installer steps individually>>).
+To run this step individually use `osx_install_ruby` tag.
```

Comment:
@pmacko1 just wondering why we remove the reference to the section `Running Installer steps individually`. Does this section get removed somewhere else?

Reply:
@laurafitzgerald nope. I think that users will look here only after reading "Run Digger Installer", containing this section.
Anyway, "running tasks individually" will be likely removed in future, some problems there (from core-team.

Reply:
@pmacko1 okay thanks for the explanation. Is there a JIRA for removing the running tasks individually or for resolving the issue so it will work? If so let's go ahead and merge this one.

Reply:
@laurafitzgerald yes, it is.
Actually, for the time being, i will comment out the "running task individually" altogether. Until the issues with it are resolved.

resolved.